appveyor.yml

```diff
@@ -1,40 +1,21 @@
 platform: Any CPU
+configuration:
+- Debug
+- Release
 image: Visual Studio 2017
 install:
 - git submodule update --init --recursive
-- ps: |
-    function gitVersion(){
-      $majorVersion = 3;
-      $minorVersion = 0;
-      $revision = 0;
-      $baseCommitHash = "d779383cb85003d6dabeb976f0845631e07bf463";
-      if ($env:APPVEYOR_REPO_BRANCH -ne 'master') {
-        $branch = "-$env:APPVEYOR_REPO_BRANCH";
-      } else {
-        $branch = "";
-      }
-
-      $build = git rev-list --count "$baseCommitHash..HEAD";
-
-      $newVersion="$majorVersion.$minorVersion.$revision.$build";
-      
-      write-host "Update appveyor build version to:$newVersion"
-      $env:ilspy_version_number=$newVersion
-      $env:appveyor_build_version="$newVersion$branch"
-      appveyor UpdateBuild -Version "$newVersion$branch"
-    }
-    gitVersion
+- ps: .\BuildTools\appveyor-install.ps1
 before_build:
 - nuget restore ILSpy.sln
 build_script:
-- msbuild ILSpy.sln /v:minimal /p:ResolveNuGetPackages=false;Configuration=Debug "/logger:%ProgramFiles%\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
-- msbuild ILSpy.sln /v:minimal /p:ResolveNuGetPackages=false;Configuration=Release "/logger:%ProgramFiles%\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
+- msbuild ILSpy.sln /v:minimal /p:ResolveNuGetPackages=false "/logger:%ProgramFiles%\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
```

Comment:
:bulb: I think you still need `;Configuration=%configuration%`

Reply:
ah, yes...

Reply:
@sharwell: msbuild imports all environment variables as properties.

This is also the reason why our `debugbuild.bat` file has to explicitly specify "/p:Platform=Any CPU" -- on some systems, a PLATFORM envvar unrelated to msbuild is otherwise causing problems.